components/Landing/Creators.tsx

```diff
@@ -43,7 +43,7 @@ export const Creators: React.FC = () => {
           >
             Creators
           </Heading>
-          <Box>
+          <Box flexDir="column" fontSize={{ base: 'md', md: '2xl' }} mb={10}>
```

Comment:
```suggestion
          <Box fontSize={{ base: 'md', md: '2xl' }} mb={10}>
```
`flexDir` doesn't do anything for `Box` component if we are not also setting `display="flex"`